keps/sig-storage/1472-storage-capacity-tracking/README.md

```diff
@@ -795,79 +846,192 @@ enhancement:
   observability if this feature?**
 
-### Dependencies
+No.
 
-Will be added before the transition to beta.
+### Dependencies
 
 * **Does this feature depend on any specific services running in the cluster?**
 
-### Scalability
+For core Kubernetes just the ones that will also run without it enabled (apiserver,
+kube-scheduler). Additional services are the CSI drivers.
+
+ * CSI driver
+   * Usage description:
+     * Impact of its outage on the feature: pods that use the CSI driver will not
+       be able to start
+     * Impact of its degraded performance or high-error rates on the
+       feature: When storage capacity information is not updated or
+       not updated often enough, then pods are either not getting
+       scheduled in cases where they could be scheduled (free capacity
+       not reported) or they get tentatively scheduled onto nodes
+       which do not have enough capacity (exhausted capacity not
+       reported). To recover from the first scenario, the driver eventually
+       needs to report capacity. To recover from the second scenario,
+       volume creation attempts will fail with "resource exhausted" and
+       other nodes have to be tried.
 
-Will be added before the transition to beta.
+### Scalability
 
 * **Will enabling / using this feature result in any new API calls?**
 
+Yes.
+
+Enabling it in apiserver and CSI drivers will cause
+`CSIStorageCapacity` objects to be created or updated. The
+number of those objects is proportional to the number of storage
+classes and number of distinct storage topology segments. For
+centralized provisioning, the number of segments is probably low. For
+distributed provisioning, the each node where the driver runs
+represents one segment, so the total number is total number of objects
+is equal to the product of "number of nodes" and "number of storage
+classes".
+
+The rate at which objects depends on how often topology and storage
+usage changes. It can estimated as:
+* creating objects for each new node and deleting them when removing a
+  node when using distributed provisioning
+* the same for adding or removing storage classes (both modes)
+* updates when volumes are created/resized/deleted (thus bounded by
+  some other API calls)
+* updates when capacity in the underlying storage system is changed
+  (usually by an administrator)
+
+Enabling it in kube-scheduler will cause it to cache all
+`CSIStorageCapacity` objects via an informer.
+
 * **Will enabling / using this feature result in introducing new API types?**
 
+Yes, `CSIStorageCapacity`.
+
 * **Will enabling / using this feature result in any new calls to cloud
   provider?**
 
+A CSI driver might have to query the storage backend more often to be
+kept informed about available storage capacity.
+
 * **Will enabling / using this feature result in increasing size or count
   of the existing API objects?**
 
+One new field gets added to `CSIDriver`.
```

Comment:
specify that it's a bool (so we get sense of size)

Reply:
Added.